integrations/langchain4j/providers/open-ai/src/test/resources/application.yaml

```diff
@@ -0,0 +1,130 @@
+#
+# Copyright (c) 2025 Oracle and/or its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+langchain4j.open-ai:
+  # ChatModel
+  chat-model:
+    api-key: api-key
+    model-name: model-name
+    base-url: base-url
+    organization-id: organization-id
+    temperature: 36.6
+    top-p: 10
+    stop: stop1,stop2,stop3
+    max-tokens: 15
+    max-completion-tokens: 20
+    presence-penalty: 0.1
+    frequency-penalty: 0.2
+    logit-bias.key1: 1
+    logit-bias.key2: 2
+    response-format: response-format
+    strict-json-schema: false
+    seed: 100
+    user: user
+    strict-tools: false
+    parallel-tool-calls: false
+    timeout: PT10M
+    max-retries: 3
+    log-requests: true
+    log-responses: true
+    custom-headers.header1: value1
+    custom-headers.header2: value2
+    # StreamingChatModel
+  streaming-chat-model:
+    api-key: api-key
+    model-name: model-name
+    base-url: base-url
+    organization-id: organization-id
+    temperature: 36.6
+    top-p: 10
+    stop: stop1,stop2,stop3
```

Comment:
I would test someting like "\n" so we are sure it doesn't get escaped or something